image/v1/types.go

```diff
@@ -267,8 +290,26 @@ type TagImportPolicy struct {
 	Insecure bool `json:"insecure,omitempty" protobuf:"varint,1,opt,name=insecure"`
 	// Scheduled indicates to the server that this tag should be periodically checked to ensure it is up to date, and imported
 	Scheduled bool `json:"scheduled,omitempty" protobuf:"varint,2,opt,name=scheduled"`
+	// ImportMode describes how to import an image manifest. Valid values are SingleManifest and ManifestList.
+	// The SingleManifest mode will result in the manifest list itself being discarded, and only one of its
+	// sub-manifests being imported, the architecture and os are then chosen in the following order of priority:
+	// 1. tag annotations; 2. control plane arch/os; 3. linux/amd64; 4. the first manifest in the list.
+	// In ManifestList mode the manifest list itself and all sub-manifests listed in it will be imported.
+	// This field is optional. SingleManifest is the default behaviour.
```

Comment:
In workload APIs, we represent the defaults by writing the defaults into the resources.  You'll need to update your openshfit-apiserver PR to specify the default values when appropriate.

Reply:
Maybe the comment should be updated... the default value for `ImportMode` is empty. An empty `ImportMode` is the same as `SingleManifest` import mode. We want this to be so for backwards compatibility.
In other words I don't think we need to always have a default set in this case because existing objects in a cluster will always be empty.

Reply:
discussed on slack, will set default to `SingleManifest` on apiserver side.

Reply:
done, marking as resolved